README.md

```diff
@@ -19,7 +19,7 @@ Manticore supports binaries of the following formats, operating systems, and
 architectures. It has been primarily used on binaries compiled from C and C++.
 Examples of practical manticore usage are also [on github](https://github.com/trailofbits/manticore-examples).
 
-- OS/Formats: Linux ELF
+- OS/Formats: Linux ELF, [Solidity](https://solidity.readthedocs.io/).
```

Comment:
we don't really analyze solidity, we analyze evm which is on the next line. i think we can omit this

Reply:
Solidity is a 'format', evm is an 'architecture', no?

Reply:
i guess so. it's an odd situation. if we added support for c files, and just had manticore call out to gcc before running on the binary, would we put C on this line?

Reply:
We wouldn't, but then Manticore would no longer be a binary analysis toolkit. The current change was meant to be succinct while still mentioning EVM analysis as the first few paragraphs of the readme. I'm okay with removing the reference to Solidity here altogether as long as we leave the ref to smart contracts in the intro